tests/parser-inference/jsfmt.spec.js

```diff
@@ -1 +1,41 @@
-run_spec(__dirname);
+const dedent = require("dedent");
+
+run_spec({
+  dirname: __dirname,
+  snippets: [
+    {
+      name: ".prettierrc in json(empty)",
+      filename: ".prettierrc",
+      code: "{}",
+    },
+    {
+      name: ".prettierrc in json",
+      filename: ".prettierrc",
+      code: dedent`
+        {printWidth: 100,
+        overrides: [
```

Comment:
This is not JSON.

Reply:
I'll add another one.

Reply:
You're aware of the trailing comma issue, right?

Reply:
I try to add `json5` comment `// comment` , seems yaml parser can't parse, so I delete that test.

trailing comma should not be a problem, it can be parsed by yaml.

Reply:
You are right, the output not right.

Reply:
Prettier doesn't support comments in `.prettierrc` if it's written in JSON.

Reply:
Trailing comma can be recognized by `cosmiconfig`, 

```console
$ cat .prettierrc
{
  "parser": "babel",
  "quoteProps": "as-needed",
}
$ node -p "require('.').resolveConfig.sync()"
{ parser: 'babel', quoteProps: 'as-needed' }
```